src/Domain/DomainHelpers.cpp

```diff
@@ -480,13 +484,22 @@ wedge_coordinate_maps(const double inner_radius, const double outer_radius,
                       const bool use_equiangular_map,
                       const double x_coord_of_shell_center,
                       const bool use_half_wedges, const double aspect_ratio,
-                      const bool use_logarithmic_map) noexcept {
+                      const bool use_logarithmic_map,
+                      const ShellPieces which_pieces) noexcept {
   const auto wedge_orientations = orientations_for_wrappings();
 
   using Wedge3DMap = CoordinateMaps::Wedge3D;
   using Halves = Wedge3DMap::WedgeHalves;
   std::vector<Wedge3DMap> wedges{};
+
   for (size_t i = 0; i < 6; i++) {
```

Comment:
Firstly, I think that what Nils suggested above would be easier to understand if formulated as
```
size_t num_pieces = blah;
for (size_t i = 6 - num_pieces; i < 6; ++i) ...
```

Secondly, is there a reason to prefer the current wedge labeling order? If the wedges were labelled in order as (+x, -x, +y, -y, +z, -z), then couldn't this loop just be `size_t i = 0; i < num_pieces; ++i` ? This would be less surprising.